src/test/java/com/chat/yourway/integration/extension/PostgresExtension.java

```diff
@@ -10,7 +10,7 @@
 public class PostgresExtension implements Extension, BeforeAllCallback, AfterAllCallback {
 
   private static final PostgreSQLContainer<?> container = new PostgreSQLContainer<>(
-      DockerImageName.parse("postgres:14.6"))
+      DockerImageName.parse("postgres:latest"))
```

Comment:
It is better to use the same database version in the container as in the working database

Reply:
I tried to run integration tests with ver. 14.6 and I got errors. Then I changed to latest and tests were passed. As for me, latest version gives me more confidence and reliability.